challenge_01/style.css

```diff
@@ -0,0 +1,95 @@
+body {
+    background-color: #E2F1E7;
+    font-family: "Figtree", sans-serif;
+    font-size: 9px;
+}
+
+.form-wrapper {
+    background-color: #ffffff;
```

Comment:
For example, this value is better to be defined as a variable. You might need to change color "white" to something that is light but not white itself, something like rgb(240, 240, 240). In your code's case, you have to change multiple places, but with CSS Custom Properties you just change it in one place.

Reply:
> For example, this value is better to be defined as a variable. You might need to change color "white" to something that is light but not white itself, something like rgb(240, 240, 240). In your code's case, you have to change multiple places, but with CSS Custom Properties you just change it in one place.

OK seems like I should learn about CSS Custom Properties 😄